tests/scene/test_path_2d.h

```diff
@@ -103,7 +103,71 @@ TEST_CASE("[SceneTree][Path2D] Curve setter and getter") {
 
 	memdelete(test_path);
 }
+	
+TEST_CASE("[SceneTree][Path2D] _edit_get_rect check") {
+	SUBCASE("Invalid curve") {
+		Path2D *test_path = memnew(Path2D);
+		Rect2 test_rect_1 = test_path->_edit_get_rect();
+		Rect2 test_rect_2 = Rect2(0, 0, 0, 0);
+        CHECK(test_rect_1 == test_rect_2);
+		memdelete(test_path);
+    }
+	SUBCASE("Valid curve with points") {
+		Path2D *test_path = memnew(Path2D);
+		Curve2D *test_curve = memnew(Curve2D);
```

Comment:
Shouldn't `Ref<Curve2D>` be used instead for consistency with other tests? This way, you won't have a need to use `memedelete()` on it to avoid a memory leak.

Reply:
Fixed!